packages/grid-pro/src/lit/column-renderer-directives.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2022 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { directive } from 'lit/directive.js';
+import { microTask } from '@vaadin/component-base/src/async';
+import { Debouncer } from '@vaadin/component-base/src/debounce';
+import { CONTENT_UPDATE_DEBOUNCER } from '@vaadin/grid/src/lit/renderer-directives.js';
+import { LitRendererDirective } from '@vaadin/lit-renderer';
+
+export class GridProColumnEditModeRendererDirective extends LitRendererDirective {
+  /**
+   * Adds the renderer callback to the grid edit column.
+   */
+  addRenderer() {
+    this.element.editModeRenderer = (root, column, model) => {
+      this.renderRenderer(root, model.item, model, column);
+    };
+  }
+
+  /**
+   * Runs the renderer callback on the grid edit column.
+   */
+  runRenderer() {
+    const grid = this.element._grid;
+
+    grid[CONTENT_UPDATE_DEBOUNCER] = Debouncer.debounce(grid[CONTENT_UPDATE_DEBOUNCER], microTask, () => {
+      grid.requestContentUpdate();
```

Comment:
Do we really need to call `requestContentUpdate()` here? As far as I can tell, the edit mode renderer is only triggered once, when user starts editing a cell. Executing this code should have no effect on the editor.